e2e/etcd_test.go

```diff
@@ -523,8 +523,8 @@ func spawnWithExpects(args []string, xs ...string) error {
 		}
 	}
 	perr := proc.Close()
-	if len(xs) == 0 && proc.LineCount() != 0 { // expect no output
```

Comment:
Can avoid ignoring the coverage tests with no lines by doing:

`if len(xs) == 0 && proc.LineCount() != noOutputLineCount {`